common/models/user.js

```diff
@@ -7,6 +7,15 @@ var config = require('../../server/config.json');
 var path = require('path');
 
 module.exports = function(User) {
+  
+  User.getVerifyOptions = function() {
+    const defaultOptions = {
```

Comment:
I would pull this up onto the User definition itself so that it's accessible for unit testing (`Unit.defaultVerifyOptions`)

Reply:
Do we really need to write test for it, since I imagine it like that.
Create user, register with user, check that first email is correct, login again with unverified user, ask to resend email, check that is the same as the value in defaultOptions and = `noreply@example.com` So this will be one hell out a test for that small change.
Our you basically want to say that just setting const defaultOptions = {} without function definition in `common/models/user.js `  would work?